### PR TITLE
Use GitHub Actions to build Docker images for multiple archs

### PR DESCRIPTION
**Pull request by wollew**
Hi, since I run owntracks on a raspberry pi and there are no container images for arm on docker hub I thought I'd give github actions a try. It works well on my fork but I'd prefer just using your repository.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    tags:
+      - v*
+
+jobs:
+  build-docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
```
[the remark this answers is not in the document]
Comment (wollew) [reply .github/workflows/main.yml]: no problem, I basically just copied this from https://github.com/marketplace/actions/docker-buildx. Fixed.

**Comment by wollew** [comment]
@jpmens: if you like the idea, I can also create a PR for owntracks/recorder, I need these ARM images there as well :-)

**Comment by linusg** [comment]
Alright let's try this!

**Comment by jpmens** [comment]
@wollew I like the idea. As soon as I see @linusg happy, you may proceed. :-)

**Comment by wollew** [comment]
> @wollew I like the idea. As soon as I see @linusg happy, you may proceed. :-)

I just saw there's a separate repo for the recorder docker images, do you want to keep it that way or merge the docker directory into the main recorder project?

**Comment by linusg** [comment]
> As soon as I see @linusg happy, you may proceed. :-)

In theory very happy, but I had to make a tweak to the workflow: https://github.com/owntracks/frontend/commit/23d73461bc3f8a1b97b4e231cfcde89dfd4b1703 (sorry about that and let's hope it works, build takes a while)

**Comment by jpmens** [comment]
@wollew IIRC there was a reason why we did this in a separate repository, but I can't remember it. If it's all the same to you, and if you don't see reasons to absolutely not do so, I'd say let's keep it separate.

But do speak out, please, if you think it would make things easier for us all if we reconcile.

**Comment by wollew** [comment]
Alright, just a quick feedback, this is not forgotten, but it seems `docker buildx` behaves a tad bit different than `docker build`, so at the end of the build it fails, not sure why:
```
#30 37.30 /usr/sbin/ot-recorder --initialize
#30 37.34 ot-recorder[415]: gcache_open: mdb_env_open: Function not implemented
```

Going to investigate as soon as I have a little bit more time for this.

**Pull request merged**